<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>pqLineWidget</class>
 <widget class="QWidget" name="pqLineWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QCheckBox" name="visible" >
     <property name="text" >
      <string>Show Line</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Point 1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QLineEdit" name="point1Z" />
     </item>
     <item row="0" column="2" >
      <widget class="QLineEdit" name="point1Y" />
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="point1X" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Point 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="point2X" />
     </item>
     <item>
      <widget class="QLineEdit" name="point2Y" />
     </item>
     <item>
      <widget class="QLineEdit" name="point2Z" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="xAxis" >
     <property name="text" >
      <string>X Axis</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="yAxis" >
     <property name="text" >
      <string>Y Axis</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="zAxis" >
     <property name="text" >
      <string>Z Axis</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>visible</tabstop>
  <tabstop>point1X</tabstop>
  <tabstop>point1Y</tabstop>
  <tabstop>point1Z</tabstop>
  <tabstop>point2X</tabstop>
  <tabstop>point2Y</tabstop>
  <tabstop>point2Z</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
